module/core/tests/Application/Form/DataTransformer/LanguageDataTransformerTest.php

```diff
@@ -64,12 +64,12 @@ public function testReverseTransform(?Language $languageValueObject, ?string $st
     }
 
     /**
-     * @expectedException \Symfony\Component\Form\Exception\TransformationFailedException
      *
-     * @expectedExceptionMessage Invalid Language "ZZ" value
+     * @expectExceptionMessage Invalid Language "ZZ" value
```

Comment:
In all application You change to: 
```suggestion
 $this->expectExceptionMessage("Invalid Language "ZZ" value");
```
Why not in this place ?